addons/strings/stringtable.xml

```diff
@@ -31,12 +32,8 @@
         </Key>
         <Key ID="STR_CBA_FORMAT_NUMBER_THOUSANDS_SEPARATOR">
             <English>,</English>
-            <Czech> </Czech>
+            <Czech>,</Czech>
             <German>.</German>
-            <Polish> </Polish>
```

Comment:
don't want to drop these

Reply:
Not quite sure how they got removed in the first place, I wasn't doing anything with those. Maybe some Tabler issue, I'll revise the XMLs manually.

Reply:
Probsbly tabler cleaning white space at EOL and then the entries being empty so they get removed.